scripts/build/build_appdeps.sh

```diff
@@ -1,14 +1,22 @@
 #!/usr/bin/env bash
+#
+# Copyright (c) .NET Foundation and contributors. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#
 
-# This file encapsulates the temporary steps to build the dotnet-compile-native command successfully
-# The AppDepSDK package is a temporary artifact until we have CoreRT assemblies published to Nuget
+set -e
 
-SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SOURCE="${BASH_SOURCE[0]}"
+while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symlink
+  DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+  SOURCE="$(readlink "$SOURCE")"
+  [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
```

Comment:
This was copy-paste from original scripts. I did the fix,  but had to touch every .sh file to make it complete :)